www/strf03/bandCamp/user_change_profile.php

```diff
@@ -0,0 +1,227 @@
+                        <input name="first_name" type="text" class="form-control" placeholder="Jméno"
+                               value="<?php echo $current_user['first_name']; ?>">
+                    </div>
+                </div>
+
+                <div class="mb-3">
+                    <label for="last_name">Příjmení</label>
+
+                    <div class="input-group">
+
+                        <input name="last_name" type="text" class="form-control" placeholder="Příjmení"
+                               value="<?php echo $current_user['last_name']; ?>">
+                    </div>
+                </div>
+
+                <div class="mb-3">
+                    <label for="phone">Telefonní číslo</label>
+
+                    <div class="input-group">
+
+                        <input name="phone" type="tel" pattern="^(\+420)? ?[1-9][0-9]{2} ?[0-9]{3} ?[0-9]{3}$}"
+                               class="form-control"
+                               placeholder="Telefonní číslo" value="<?php echo $current_user['phone']; ?>">
+                    </div>
+                </div>
+
+                <div class="mb-3">
+                    <label for="music_genres">Co máš rád? (Podrž ctrl pro označení více)</label>
+                    <select name="music_genres[]" class="form-control" multiple="multiple">
+                        <?php foreach ($music_genres as $music_genre): ?>
+                            <option value="<?php echo $music_genre['music_genre_id']; ?>"><?php echo $music_genre['music_genre_name']; ?></option>
+                        <?php endforeach; ?>
+                    </select>
+
+                </div>
+
+                <div class="mb-3">
+                    <label for="instruments">Na co hraješ? (Podrž ctrl pro označení více)</label>
+                    <select name="instruments[]" class="form-control" multiple="multiple">
+                        <?php foreach ($instruments as $instrument): ?>
+                            <option value="<?php echo $instrument['instrument_id']; ?>"><?php echo $instrument['instrument_name']; ?></option>
+                        <?php endforeach; ?>
+                    </select>
+
+                </div>
+
+                <div class="mb-3">
+                    <label for="district">Kraj</label>
+                    <select name="district" class="custom-select d-block w-100">
+                        <option value="">Vyber kraj...</option>
+                        <?php foreach ($districts as $name): ?>
+                            <option value="<?php echo $name ?>" <?php echo $name == $current_user['district'] ? 'selected' : '' // todo dát všude?> > <?php echo $name ?> </option>
+                        <?php endforeach; ?>
+                    </select>
+                </div>
+
+                <div class="mb-3">
+                    <label for="password">Heslo</label>
+                    <input name="password" type="password" class="form-control" placeholder="Password">
```

Comment:
zmena hesla spolecne s profilem? a navic je to povinne?

Reply:
Smazal jsem to i z úpravy profilu kapely. Změnu hesla vyřeším jinak na samostatném formuláři.